e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -1024,3 +1025,71 @@ describe.skip("issue 49142", () => {
       .and("contain", "Embeddable dashboard");
   });
 });
+
+describe("issue 8490", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    createDashboardWithQuestions({
+      dashboardDetails: {
+        name: "Dashboard with a parameter",
+        enable_embedding: true,
+      },
+      questions: [
+        {
+          name: "Line chart",
+          query: {
+            "source-table": PRODUCTS_ID,
+            aggregation: [["count"]],
+            breakout: [
+              ["field", PRODUCTS.CATEGORY, { "base-type": "type/Text" }],
+              [
+                "field",
+                PRODUCTS.CREATED_AT,
+                { "base-type": "type/DateTime", "temporal-unit": "month" },
+              ],
+            ],
+          },
+          visualization_settings: {
+            "graph.dimensions": ["CREATED_AT", "CATEGORY"],
+            "graph.metrics": ["count"],
+          },
+          display: "bar",
+        },
+      ],
+    }).then(({ dashboard, questions: [question] }) => {
+      cy.wrap(dashboard.id).as("dashboardId");
+    });
+  });
+
+  it("static embeddings should respect `#locale` hash in the URL (metabase#8490)", () => {
+    cy.get("@dashboardId").then(dashboardId => {
+      visitEmbeddedPage(
+        {
+          resource: { dashboard: dashboardId },
+          params: {},
+        },
+        {
+          additionalHashOptions: {
+            locale: "ko",
+          },
+        },
+      );
+    });
+
+    cy.findByTestId("embed-frame").within(() => {
+      // X-axis labels: Jan 2023
+      cy.findByText("1월 2023").should("be.visible");
+
+      // PDF export
+      cy.findByText("PDF로 내보내기").should("be.visible");
+
+      // Powered by
+      cy.findByText("제공:").should("be.visible");
+
+      // Aggregation "count"
+      cy.findByText("카운트").should("be.visible");
```

Comment:
This assertion on the chart label (aggregation) will fail without [the BE patch](https://github.com/metabase/metabase/pull/49441/commits/83d8d1358ce30019291b5a0b17d69528f196227a).

You can try to revert that commit and run this test again, and the test should fail (I've tested that already)

Reply:
I was going to suggest if we can use a question/dashboard already present in the test db, but I guess we need the aggregation specifically for this assertion

Reply:
I think the difference would be minimal, wouldn't it? With a few less requests for creating a question and a dashboard. That might save like 150ms on Cypress on my Machine (could be longer on CI I guess), but I couldn't find a compatible dashboard that alreay exist, so I guess it's fine to do so.

And we've been creating new resources like this in a lot of tests anyway.

Why do you think we should use existing data? Was this concern raised before?

Reply:
I just find the code to create the questions un-necessary if we already have the questions pre-made, but it's probably personal preference.
Creating them ad-hoc probably makes it clearer what we're opening

Reply:
there are pros and cons to using premade questions.
**Pros**
- Faster. Maybe this could be a huge boost imagine that we have a pool of questions and dashboards that maybe work for like 80% of the use cases. Then we could potentially reduce the time to run tests, hopefully, by a lot.

**Cons**
- Dependent on the DB snapshot
- Less clear what question we're referring to. Compare to the immediate `questionDetails` that you could read in each test.

Again unless we know exactly what we're trying to achieve here. I'd still prefer creating new resources for redability.